Patch filenames of third-party sources to include version numbers.
--- cmake/ThirdPartyTriangle.cmake.orig	2020-07-10 11:22:33.000000000 +0100
+++ cmake/ThirdPartyTriangle.cmake	2020-07-10 11:22:52.000000000 +0100
@@ -25,7 +25,7 @@
         EXTERNALPROJECT_ADD(
             triangle-1.6
             PREFIX ${TPSRC}
-            URL ${TPURL}/triangle.zip
+            URL ${TPURL}/triangle-1.6.zip
             URL_MD5 357cb7107f51f3f89940c47435d4fa49
             STAMP_DIR ${TPBUILD}/stamp
             DOWNLOAD_DIR ${TPSRC}
--- cmake/ThirdPartyTetGen.cmake.orig	2020-07-10 11:23:15.000000000 +0100
+++ cmake/ThirdPartyTetGen.cmake	2019-11-19 16:55:14.000000000 +0000
@@ -25,7 +25,7 @@
         EXTERNALPROJECT_ADD(
             tetgen-1.5
             PREFIX ${TPSRC}
-            URL ${TPURL}/tetgen.zip
+            URL ${TPURL}/tetgen-1.5.zip
             URL_MD5 6d62e63f9b1e7a8ce53d5bc87e6a0a09
             STAMP_DIR ${TPBUILD}/stamp
             DOWNLOAD_DIR ${TPSRC}
